plugins/module_utils/network/sonic/facts/ospfv2_interfaces/ospfv2_interfaces.py

```diff
@@ -0,0 +1,197 @@
+                        if ospf_int:
+                            ospf_attributes = []
+                            for address in ospf_int.get('if-addresses', []):
+                                attr = {}
+                                addr = address.get('address')
+                                bfd = address.get('enable-bfd')
+                                if bfd:
+                                    cfg = bfd.get('config')
+                                    if cfg:
+                                        bfd_cfg = {}
+                                        self.update_dict(bfd_cfg, 'enable', cfg.get('enabled'))
+                                        self.update_dict(bfd_cfg, 'bfd_profile', cfg.get('bfd-profile'))
+                                        self.update_dict(ospf_config, 'bfd', bfd_cfg)
+                                config = address.get('config')
+                                md_authentications = address.get('md-authentications')
+                                if config:
+                                    self.update_dict(attr, 'area_id', config.get('area-id'))
+                                    self.update_dict(attr, 'authentication_type', config.get('authentication-type'))
+                                    self.update_dict(attr, 'cost', config.get('metric'))
+                                    self.update_dict(attr, 'hello_interval', config.get('hello-interval'))
+                                    self.update_dict(attr, 'mtu_ignore', config.get('mtu-ignore'))
+                                    self.update_dict(attr, 'priority', config.get('priority'))
+                                    self.update_dict(attr, 'retransmit_interval', config.get('retransmission-interval'))
+                                    self.update_dict(attr, 'transmit_delay', config.get('transmit-delay'))
+
+                                    if 'authentication-key' in config:
+                                        attr['auth_pwd'] = {}
+                                        self.update_dict(attr['auth_pwd'], 'pwd', config.get('authentication-key'))
+                                        self.update_dict(attr['auth_pwd'], 'encrypted', config.get('authentication-key-encrypted'))
+
+                                    if 'dead-interval-minimal' in config:
+                                        dead_interval_minimal = config.get('dead-interval-minimal')
+                                        if dead_interval_minimal:
+                                            self.update_dict(attr, 'hello_multiplier', config.get('hello-multiplier'))
+                                        else:
+                                            self.update_dict(attr, 'dead_interval', config.get('dead-interval'))
+                                    elif 'dead-interval' in config:
+                                        self.update_dict(attr, 'dead_interval', config.get('dead-interval'))
```

Comment:
Could be simplified as:
```suggestion
                                    if config.get('dead-interval-minimal'):
                                        self.update_dict(attr, 'hello_multiplier', config.get('hello-multiplier'))
                                    else:
                                        self.update_dict(attr, 'dead_interval', config.get('dead-interval'))
```